eth/catalyst/simulated_beacon_test.go

```diff
@@ -96,7 +96,7 @@ func TestSimulatedBeaconSendWithdrawals(t *testing.T) {
 	defer subscription.Unsubscribe()
 
 	// generate some withdrawals
-	for i := 0; i < 20; i++ {
```

Comment:
This test case should have been like this originally.  The withdrawal queue has a max size of 10.

Previously, this passed because there was no contention between adding withdrawals and committing blocks.  Now that access to withdrawals is gated behind a mutex, the contention causes all withdrawals in the loop to be added without simultaneously sealing blocks.